extension/k8s/values.yaml

```diff
@@ -0,0 +1,28 @@
+fullnameOverride: 'ctp-adyen-integration-service-extension'
+
+image:
+  repository: 'eu.gcr.io/professionalserviceslabs/ctp-adyen-integration-extension'
+  tag: '1.0'
+
+service:
+  type: 'NodePort'
+
+# More on resources:
+# https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/resource-qos.md
+# https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
+# https://cloud.google.com/compute/docs/machine-types
+
+ingress:
+  enabled: true
+  annotations:
+    kubernetes.io/ingress.class: 'gce'
+    kubernetes.io/ingress.allow-http: 'false'
+
+livenessProbe:
+  httpGet:
+    path: ''
+
+
+readinessProbe:
```

Comment:
Do we have a TODO for that?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/issues/50